plugins/catalog-backend/src/database/DefaultProcessingDatabase.ts

```diff
@@ -356,6 +358,18 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
         this.options.logger.warn(
           `Detected conflicting entityRef ${entityRef} already referenced by ${conflictingKey} and now also ${locationKey}`,
         );
+        if (this.options.conflictEventBroker) {
+          await this.options.conflictEventBroker?.publish({
+            topic: 'conflicts',
```

Comment:
should this be "catalog.conflict"

Reply:
Sure - is the dot syntax a convention?

Reply:
I think so it is a convention.. although I am not sure. I believe the github ones are `github.${eventName}` e.g. `github.push`

@pjungermann would you have an answer for that?